CI/windows/03_package_obs.ps1

```diff
@@ -123,7 +123,7 @@ function Package-OBS-Standalone {
         $VersionString = "${GitTag}-${GitHash}"
     }
 
-    $FileName = "${ProductName}-${VersionString}"
+    $FileName = "obs-studio-windows-${VersionString}"
```

Comment:
Prefer that the OBS Studio version string is immediately after "obs-studio" to associate the two pieces of information.

```
obs-studio-${VERSION_STRING}-windows
```

Reply:
fixed